_pages/governance.md

```diff
@@ -454,16 +452,16 @@ Center for Reproducible Neuroscience
 blog](http://reproducibility.stanford.edu/blog/){:target="_blank"} provides tutorials and
 community survey results.
 
-All BIDS community members are required to follow the [BIDS code of
-conduct](https://github.com/bids-standard/bids-specification/blob/master/CODE_OF_CONDUCT.md){:target="_blank"}.
-Please contact Franklin Feingold at [ffein@stanford.edu](mailto:ffein@stanford.edu)
+All BIDS community members are required to follow the
+[BIDS code of conduct](https://github.com/bids-standard/bids-specification/blob/master/CODE_OF_CONDUCT.md){:target="_blank"}.
+Please contact the BIDS maintainers at [bids.maintenance+coc@gmail.com](mailto:bids.maintenance+coc@gmail.com)
```

Comment:
@Remi-Gau I assume I need to add some "coc" and "question" category to the bids.maintenance Gmail account now? Do you think this is a good idea btw?

Reply:
Note: I added re-directs for "+coc" and "+question" to the Gmail account.

Reply:
anything that can do triage automatically should help future us and future maintainers, so go for.